sql/2019/04_Media/04_12.sql

```diff
@@ -0,0 +1,17 @@
+/*
+standard SQL
+04_12
+112 GB
+
+this gets the requests whose servers add a "Vary: User-Agent" or "Vary: Accept" 
+(not a lot of hits here... many have Vary: Accept-Encoding)
+
+*/
+
+
+select url, NET.HOST(url) host, respOtherHeaders
+
+
+from `summary_requests.2019_07_01_mobile`
+where respOtherHeaders like "%Vary:%User-Agent%" OR respOtherHeaders like "%Vary:%Accept%"
```

Comment:
The `Vary` header is extracted into the specific `resp_vary` field, IIRC that would omit it from the "other" field. You should also convert the value to lowercase to maximize detections:

```sql
WHERE LOWER(resp_vary) IN ('user-agent', 'accept')
```